ts/features/idpay/initiative/details/components/InitiativeCardComponent.tsx

```diff
@@ -23,133 +24,157 @@ type Props = {
+      ? (remainingAmount / amount) * 100.0
+      : 100.0;
+  return (
+    <View style={styles.cardContainer} testID={"card-component"}>
+      {/* top part */}
+      <View style={styles.topCardSection}>
+        <View style={styles.bonusLogoContainer}></View>
+        <VSpacer size={8} />
+        <H1>{initiativeName}</H1>
+        <Body>{/* the ministry would go here */}</Body>
+        <VSpacer size={8} />
+        <View style={styles.bonusStatusContainer}>
+          <Badge style={styles.badge}>
+            <LabelSmall color="white">
               {I18n.t(
-                "idpay.initiative.details.initiativeCard.availableAmount"
+                `idpay.initiative.details.initiativeCard.statusLabels.${status}`
               )}
-            </H5>
-          </View>
+            </LabelSmall>
+          </Badge>
           <HSpacer size={8} />
-          <View>
-            <Text
-              bold={true}
-              style={[
-                !isInitiativeConfigured ? styles.consumedOpacity : {},
-                styles.amount
-              ]}
-            >
-              {formatNumberAmount(toBeRepaidAmount, true)}
-            </Text>
-            <H5 color="white">
-              {I18n.t("idpay.initiative.details.initiativeCard.toRefund")}
-            </H5>
-          </View>
+          <LabelSmall color="bluegreyDark">
+            {I18n.t(
+              `idpay.initiative.details.initiativeCard.${
+                isInitiativeConfigured ? "validUntil" : "expiresOn"
+              }`,
+              {
+                expiryDate: dateString
+              }
+            )}
+          </LabelSmall>
         </View>
       </View>
-      <View>
-        <View
-          style={styles.bottomInitiativeIcon}
-          accessible={true}
-          importantForAccessibility={"no-hide-descendants"}
-          accessibilityElementsHidden={true}
-        >
-          <Image source={bonusVacanzeWhiteLogo} style={styles.logo} />
+
+      <VSpacer size={48} />
+      {/* bottom part */}
+
+      <View style={styles.bottomCardSection}>
+        <View style={styles.alignCenter}>
+          <LabelSmall color="bluegreyDark" weight="Regular">
+            {I18n.t("idpay.initiative.details.initiativeCard.availableAmount")}
+          </LabelSmall>
+          <H1 style={!isInitiativeConfigured ? styles.consumedOpacity : {}}>
+            {formatNumberAmount(remainingAmount, true)}
+          </H1>
+          <VSpacer size={8} />
+          <BonusPercentageSlider
+            isGreyedOut={isInitiativeConfigured}
+            percentage={remainingBonusAmountPercentage}
+          />
+        </View>
+        <View style={styles.alignCenter}>
+          <LabelSmall color="bluegreyDark" weight="Regular">
+            {I18n.t("idpay.initiative.details.initiativeCard.toRefund")}
+          </LabelSmall>
+          <H1 style={!isInitiativeConfigured ? styles.consumedOpacity : {}}>
+            {formatNumberAmount(toBeRepaidAmount, true)}
```

Comment:
```suggestion
            {`${formatNumberAmount(toBeRepaidAmount, false)} €`}
```
Inverted currency symbol

Reply:
this and other proposed changes are adressed in https://github.com/pagopa/io-app/pull/4436/commits/dbb109705f14dcfcb7238e3c345a7878d0784bac